news_events/etl/sloan_exec_news.py

```diff
@@ -0,0 +1,192 @@
+"""ETL for blog/news from Sloan School of Management Executive Education"""
+
+import html
+import logging
+import re
+from urllib.parse import urlencode, urljoin
+
+import requests
+from bs4 import BeautifulSoup as Soup
+
+from news_events.constants import FeedType
+from news_events.etl.utils import tag_text
+
+log = logging.getLogger(__name__)
+
+SLOAN_EXEC_TITLE = "MIT Sloan Executive Education"
+SLOAN_EXEC_BASE_URL = "https://exec.mit.edu/"
+SLOAN_EXEC_BLOG_URL = urljoin(SLOAN_EXEC_BASE_URL, "/s/blog")
+SLOAN_EXEC_ARTICLE_PREFIX_URL = urljoin(SLOAN_EXEC_BASE_URL, "/s/blog-post/")
+SLOAN_EXEC_POST_URL = urljoin(
+    SLOAN_EXEC_BASE_URL, "/s/sfsites/aura?r=15&other.CMSContent.fetchTopicPage=1"
+)
+SLOAN_EXEC_POST_DATA = (
+    urlencode(
+        {
+            "message": {
+                "actions": [
+                    {
+                        "id": "111;a",
+                        "descriptor": "apex://CMSContentController/ACTION$fetchTopicPage",
+                        "callingDescriptor": "markup://c:CMS_BlogPostList",
+                        "params": {
+                            "contentType": "Blog_Post",
+                            "topicName": "",
+                            "pageSize": 20,
+                            "pageNumber": 0,
+                            "sortFields": ["DateTime1"],
+                            "AscDesc": "DESC",
+                        },
+                    }
+                ]
+            },
+            "aura.context": {
+                "mode": "PROD",
+                "fwuid": "FWUID_VAR",
+                "app": "siteforce:communityApp",
+                "loaded": {
+                    "APPLICATION@markup://siteforce:communityApp": "APPID_VAR",
+                    "COMPONENT@markup://instrumentation:o11ySecondaryLoader": "",
+                },
+                "dn": [],
+                "globals": {},
+                "uad": False,
+            },
+            "aura.pageURI": "/s/blog",
+            "aura.token": None,
+        }
+    )
+    .replace("None", "null")
+    .replace("False", "false")
+    .replace("+", "")
+    .replace("%27", "%22")
+)
+
+
+def extract() -> dict:
+    """
+    Extract JSON from Sloan School of Management
+
+    Returns:
+        dict: JSON data from Sloan blog post request
+    """
+    session = requests.Session()
+    content = str(session.get(SLOAN_EXEC_BLOG_URL).content)
+    fwuid = re.findall(r"fwuid%22%3A%22([^%]+)%22", content)[0]
+    appId = re.findall(r"siteforce%3AcommunityApp%22%3A%22([^%]+)%", content)[0]
```

Comment:
I inspected the site to see if there was a better structured way to get this info but it does seem like whatever we do would end up being hacky.